misc/maven-resolver/src/main/java/module-info.java

```diff
@@ -1,13 +0,0 @@
-module io.ballerina.maven.resolver {
```

Comment:
Removing the `module-info.java` here is intentional right?

Reply:
Yes Apache Maven Resolver has issues with JPMS
https://stackoverflow.com/questions/60915727/java-11-unnamed-module-issue-when-using-maven-model